cypress/e2e/learning_store/manageAccessToLearningstore.feature

```diff
@@ -5,32 +5,25 @@ Feature: Learning store - Activating and deactivating access for students
 
     @stable_test
     Scenario Outline: Admin activates and deactivates students access to Learning store
-    # Admin deactivates students access to Learning store - pre-condition to set the needed configuration
-        Given I am logged in as a '<admin>' at '<namespace>'
-        When I go to administration page
-        When I go to school administration
-        When I click the checkbox to disable students access to learning store
-        When I click on admin setting save button
-        Then I see checkbox to enable students access to learning store is unchecked
-    # Student doesn't see link to Learning store in menu
-        Given I am logged in as a '<student>' at '<namespace>'
-        Then I do not see Learning Store in side bar
     # Admin activates students access to Learning store
         Given I am logged in as a '<admin>' at '<namespace>'
         When I go to administration page
         When I go to school administration
+        When I click on general settings panel
         When I click the checkbox to enable students access to learning store
         When I click on admin setting save button
```

Comment:
Same here before clicking on save btn. Can you check is toggle really switched on or not?

Reply:
see above